package.json

```diff
@@ -2,7 +2,14 @@
   "name": "media-chrome",
   "version": "0.5.3",
   "description": "Custom elements (web components) for making audio and video player controls that look great in your website or app.",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/index.js",
+  "browser": "dist/iife/index.js",
+  "unpkg": "dist/iife/index.js",
```

Comment:
this can be removed or pointed at `dist/index.js` if we wanted `type=module` being the default for `<script>`